readme.md

```diff
@@ -0,0 +1,161 @@
+# Remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+**[remark][]** is a tool that transforms markdown with plugins.
+These plugins can inspect and change your markup.
+You can use remark on the server, the client, CLIs, deno, etc.
+This plugin for Visual Studio Code lets you format and lint markdown files using
+remark.
+
+## Contents
+
+*   [When should I use this?](#when-should-i-use-this)
+*   [Use](#use)
+    *   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## When should I use this?
+
+If your project is checked using [`remark-cli`][remark-cli] and you use Visual
+Studio Code to edit markdown files, it’s recommended to use this extension.
+
+## Use
+
+To start using this for a project, [`remark-cli`][remark-cli] needs to be
+configured first.
+For example, install `remark-cli` and `remark-preset-lint-recommended`:
+
+```sh
+npm install remark-cli remark-preset-lint-recommended
+```
+
+Now create a file named `.remarkrc.json` with the following content:
```

Comment:
While `.json` isn’t bad, it feels more of a personal convention to use it here. This VS Code extension highlights `.remarkrc` as well. Should it be explained why this file is named that way?

Reply:
I had to pick something. I think `.remarkrc.json` is understood by most people.

I don’t think we should suggest `.remarkrc` just because this extension supports it. I think it’s best to always use an extension. `.eslintrc` without extension has even been deprecated.

I don’t think we need to explain why the file is named that way. It’s a common convention.

Reply:
> I don’t think we need to explain why the file is named that way. It’s a common convention.

I think people will come into this extension not knowing what remark is. Or knowing that RC files exist.